src/main/java/net/ccbluex/liquidbounce/injection/mixins/minecraft/item/MixinHeldItemRenderer.java

```diff
@@ -170,4 +173,13 @@ private void transformLegacyBlockAnimations(AbstractClientPlayerEntity player, f
         }
     }
 
+    @Redirect(method = "updateHeldItems", at = @At(value = "INVOKE", target = "Lnet/minecraft/client/network/ClientPlayerEntity;getMainHandStack()Lnet/minecraft/item/ItemStack;"))
```

Comment:
Please do not use `@Redirect`

Reply:
I doubt any mod will creat a mixin to the exact same point but sure I'll change it to `ModifyExpressionValue`.